api/v1/managementcluster_types.go

```diff
@@ -27,6 +27,30 @@ type ManagementClusterSpec struct {
 	// Valid examples are: "0.0.0.0:31000", "example.com:32000", "[::1]:32500"
 	// +optional
 	Address string `json:"address,omitempty"`
+
+	// TLS provides options for configuring how Managed Clusters can establish an mTLS connection with the Management Cluster.
+	// +optional
+	TLS TLS `json:"tls,omitempty"`
+}
+
+type TLS struct {
+	// secretName indicates the name of the secret in the tigera-operator namespace that contains a certificate bundle which should be used to serve voltron's tunnel.
```

Comment:
I think we try in most of our docs to not mention the name voltron.
How about this:

```
// SecretName indicates the name of the secret in the tigera-operator namespace that contains the private key and certificate that the management cluster uses when it listens for incoming connections.
```